packages/cli/src/oclif/commands/invoke.js

```diff
@@ -519,7 +516,7 @@ class InvokeCommand extends BaseCommand {
       );
     }
     const authData = await this.promptForAuthFields(
-      appDefinition.authentication.fields,
+      appDefinition.authentication.inputFields,
```

Comment:
`zapier invoke` is still gaining adoption. It would hurt if it starts breaking. So I'd like the `invoke` command to continue working on previous versions of platform-core for at least some time. Maybe we could do something like:

```suggestion
      appDefinition.authentication.fields || appDefinition.authentication.inputFields,
```

And add this back at line 336-338:

```js
      if (field.computed) {
        continue;
      }
```